internal/js/modules/k6/browser/common/locator.go

```diff
@@ -685,6 +685,28 @@ func (l *Locator) press(key string, opts *FramePressOptions) error {
 	return l.frame.press(l.selector, key, opts)
 }
 
+// Press Sequentially focuses the element and then sends a keydown, keypress/input, and keyup event for each character in the text.
+// for special keys -- locator.press() should be used
+func (l *Locator) PressSequentially(text string, opts *FramePressOptions) error {
+	l.log.Debugf(
+		"Locator:PressSequentially", "fid:%s furl:%q sel:%q text:%q opts:%+v",
+		l.frame.ID(), l.frame.URL(), l.selector, text, opts,
+	)
+	_, span := TraceAPICall(l.ctx, l.frame.page.targetID.String(), "locator.pressSequentially")
+	defer span.End()
+
+	opts.Strict = true
+	for _, char := range text {
+		if err := l.frame.press(l.selector, string(char), opts); err != nil {
+			err := fmt.Errorf("pressing character %q: %w", char, err)
```

Comment:
We want to know the error's origin (pressSequentially) when it happens:

```suggestion
			err := fmt.Errorf("pressing character %q sequentially: %w", char, err)
```